docs/svelte-testing-library/api.mdx

```diff
@@ -27,7 +27,7 @@ tells Svelte to apply any new changes to the DOM.
 ```js
 import {render} from '@testing-library/svelte'
 
-const {results} = render(YourComponent, {ComponentOptions}, {RenderOptions})
+const results = render(YourComponent, {ComponentOptions}, {RenderOptions})
```

Comment:
Can this be renamed to `view`?
The reason for this change is because that's how we encourage it via the ESLint plugin.
https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/render-result-naming-convention.md 

```suggestion
const view = render(YourComponent, {ComponentOptions}, {RenderOptions})
```